src/Hangfire.PostgreSql/PostgreSqlStorageOptions.cs

```diff
@@ -39,6 +40,7 @@ public PostgreSqlStorageOptions()
     {
       QueuePollInterval = TimeSpan.FromSeconds(15);
       InvisibilityTimeout = TimeSpan.FromMinutes(30);
+      SlidingInvisibilityTimeout = null;
```

Comment:
Default of nullable field is already null. No need to set it explicitly if it's the same.

Reply:
Removed this and replaced with boolean UseSlidingInvisibilityTimeout